com.wso2telco.dep.hub.common/com.wso2telco.dep.hub.common.synapse/src/main/synapse-config/sequences/commonFault.xml

```diff
@@ -731,19 +663,45 @@
             </payloadFactory>
         </default>
     </switch>
+
+    <filter regex="^(?i)(yes|no)$" source="$ctx:BACKEND_ERROR_SET" xmlns:ns="http://org.apache.synapse/xsd">
```

Comment:
Please change this to true|false. The property should be BACKEND_ERROR not BACKEND_ERROR_SET.